src/main/java/digital/slovensko/autogram/ui/cli/CliApp.java

```diff
@@ -21,29 +16,26 @@ public static void start(CommandLine cmd) {
         var ui = new CliUI();
 
         try {
-            var params = new CliParameters(cmd);
-            var autogram = new Autogram(ui, false, params.getDriver() != null ?
-                        () -> Collections.singletonList(params.getDriver())
-                        : new DefaultDriverDetector("", false),
-                    params.getSlotId(), params.getTspSource());
+            var settings = CliSettings.fromCmd(cmd);
```

Comment:
Toto som dost prekopal, ale neviem to uplne spustit :sweat_smile: cize toto by si mohol skusit pretestovat? @celuchmarek 

V podstate som uplne vyrazil tie CliParameters a vyrobil len CliSettings, ktora cosi naviac dava oproti beznemu UserSettings.